ngboost/distns/poisson.py

```diff
@@ -43,15 +39,15 @@ def __init__(self, params):
         self.dist = dist(mu=self.mu)
 
     def fit(Y):
-        assert(np.equal(np.mod(y, 1), 0).all), "All Poisson target data must be discrete integers"
+        assert(np.equal(np.mod(Y, 1), 0).all), "All Poisson target data must be discrete integers"
```

Comment:
```suggestion
        assert(np.equal(np.mod(Y, 1), 0).all()), "All Poisson target data must be discrete integers"
```
My bad. missed `.all()` in first suggestion